readme.md

```diff
@@ -26,12 +26,12 @@ $ npm install ow
 
 ## Usage
 
-*If you use CommonJS, you need to import is as `const {default: ow} = require('ow')`.*
+_If you use CommonJS, you need to import is as `const {default: ow} = require('ow')`._
```

Comment:
Don't make unrelated changes

Reply:
Sorry, linter was on. Should be good now.